x/emissions/keeper/queryserver/query_server_inclusions_test.go

```diff
@@ -0,0 +1,53 @@
+package queryserver_test
+
+import (
+	"github.com/allora-network/allora-chain/x/emissions/types"
+)
+
+func (s *QueryServerTestSuite) TestGetCountInfererInclusionsInTopic() {
+	s.CreateOneTopic()
+	ctx := s.ctx
+	queryserver := s.queryServer
+	topicId := uint64(1)
+	inferer := "allo10es2a97cr7u2m3aa08tcu7yd0d300thdct45ve"
```

Comment:
Can we make a `testutil.RandomAddress()` func?  Or if this value needs to be exactly this because of test setup, can this be made into a testutils variable?

Reply:
this should be s.addrsStr[0] instead

every test suite has a handful of addrs on hand for testing with. Nothing needs to be hardcoded ever